src/components/App.jsx

```diff
@@ -58,13 +58,24 @@ export const App = () => {
           <>
             <BarCenter>
               <MuiCozyTheme>
-                <Typography variant="h5">{client.appMetadata.slug}</Typography>
+                <Typography variant="h5">{client.appMetadata.slug} </Typography>
```

Comment:
I don't like relying of this kind of stuff. Linter can easily remove those trailing space. Why is is needed? Can't we add a margin if really needed?

Reply:
Ce n'est pas voulu, je me rends compte que le linter ne réagit pas dans ce cas précis